images/src/ContentReactor.Images/ContentReactor.Images.Services/ImagesService.cs

```diff
@@ -97,8 +104,8 @@ public async Task<(CompleteAddImageNoteResult result, string previewUri)> Comple
                 }
 
                 // validate the image is in an acceptable format
-                var validationResult = ImageValidatorService.ValidateImage(rawImage);
-                if (!validationResult.isValid)
+                var (isValid, mimeType) = ImageValidatorService.ValidateImage(rawImage);
```

Comment:
We typically avoid using tuples in public interfaces.   This seems clear enough to me, but we don't gain very much from it.   We may want to consider keeping the previous form to ensure that we don't confuse and alienate those unfamiliar with tuples and object destructuring.